File: fabric-like/src/main/java/net/mobmincer/fabriclike/MobMincerFabricLike.java
package net.mobmincer.fabriclike;

import net.fabricmc.api.EnvType;
import net.fabricmc.api.Environment;
import net.mobmincer.MobMincer;

public class MobMincerFabricLike
{
    public static void init() {
        MobMincer.init();
    }

    @Environment(EnvType.CLIENT)
    public static void initClient() {
        MobMincer.initClient();
    }
}

File: fabric-like/src/main/java/net/mobmincer/fabriclike/MobMincerFabricLike.kt
package net.mobmincer.fabriclike

import fuzs.forgeconfigapiport.api.config.v3.ForgeConfigRegistry
import net.fabricmc.api.EnvType
import net.fabricmc.api.Environment
import net.mobmincer.MobMincer
import net.mobmincer.core.config.MobMincerConfig
import net.neoforged.fml.config.ModConfig

object MobMincerFabricLike {
    @JvmStatic
    fun init() {
        ForgeConfigRegistry.INSTANCE.register(MobMincer.MOD_ID, ModConfig.Type.COMMON, MobMincerConfig.SPEC)
        MobMincer.init()
    }

    @JvmStatic
    @Environment(EnvType.CLIENT)
    fun initClient() {
        MobMincer.initClient()
    }
}
